topo_processor/file_system/file_system_local.py

```diff
@@ -0,0 +1,39 @@
+import os
+from shutil import copyfile  # not async to be replaced
+
+import pystac
+
+from topo_processor.stac import Asset, Collection, Item
+from topo_processor.util import multihash_as_hex, write_stac_metadata
+
+from .file_system import FileSystem
+
+
+class FileSystemLocal(FileSystem):
+    name = "file.system.local"
+
+    async def read(self):
+        pass
+
+    async def write(self, collection: Collection, target: str):
+        if not os.path.isdir(os.path.join(target, collection.title)):
+            os.makedirs(os.path.join(target, collection.title))
+        await super().write(collection, target)
+
+    async def write_asset(self, asset: Asset, target: str):
+        asset.properties["file:checksum"] = await multihash_as_hex(asset.source_path)
+        copyfile(asset.source_path, os.path.join(target, asset.target))
```

Comment:
we can have a local read and remote write so this won't work

Reply:
use get_file_system(asset.source_path) as use this as the write